versions.tf

```diff
@@ -1,12 +1,13 @@
 terraform {
+  required_version = ">= 1.2.5"
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.27.0"
+      version = ">= 4.0.0"
```

Comment:
Is this downgrade intentional?

Reply:
👋🏼 Yes, it is. Historically we used to use a generic version in our modules, which happened to be `4.27.0` when `cloud-platform-environments` namespaces' `versions.tf` pointed to `4.27.0` (i.e. it matched). Going forward, Cloud Platform provided modules will only need to bump the _major_ version and we can use the namespaces' `versions.tf` to pin a provider version.

It gets rid of some toil (i.e. updating 4.27.0 to the next minor release) that we haven't always kept on top of.